jenkins/Pipeline.gy

```diff
@@ -25,6 +39,14 @@ pipeline {
     }
     post {
         success {
+            // This symlink somehow allows the Cobertura plug-in to find the
+            // sources referenced in the coverage report.  However, the presence
+            // of this link inside the container seems to confuse the coverage
+            // data collection, so we create it here and then remove it after
+            // generating the report. (We use the --force option just in case
```

Comment:
You probably shouldn't *say* you're using `--force` when you're *actually* using `-f` ... or perhaps you should actually use `--symbolic --no-target-directory --force`?

Reply:
I addressed this in #2938.